src/clj_discord_bot/commands/evangelize.clj

```diff
@@ -0,0 +1,20 @@
+(ns clj-discord-bot.commands.evangelize
+  (:require [clj-discord.core :as discord]))
+
+(defonce lib (keys (ns-publics 'clojure.core)))
+(defonce adjectives (clojure.string/split-lines (slurp "assets/adjectives")))
+
+(defn docstring [fn-name]
+  (:doc (meta (resolve (symbol fn-name)))))
+
+(defn get-propaganda [type data]
+  (discord/answer-command data
+                          (get data "content")
+                          (let [fn-name (rand-nth lib)]
```

Comment:
I'd consider moving `(rand-nth adjectives)` into the `let` as well, or getting rid of the `let` altogether.